tests/sles4sap/publiccloud/qesap_ansible.pm

```diff
@@ -35,7 +36,48 @@ sub run {
     # skip ansible deployment in case of reusing infrastructure
     unless (get_var('QESAP_DEPLOYMENT_IMPORT')) {
         my @ret = qesap_execute(cmd => 'ansible', timeout => 3600, verbose => 1);
-        die("Ansible deploymend FAILED. Check 'qesap*' logs for details.") if $ret[0] > 0;
+        if ($ret[0]) {
+            if (qesap_file_find_string(file => $ret[1], search_string => 'Timed out waiting for last boot time check')) {
```

Comment:
is it similar code from https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/qesapdeployment/deploy.pm#L25

If it is so, could you please try to move the duplicated code to a dedicated method in lib/qesapdeploy.pm

Reply:
> is it similar code from https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/qesapdeployment/deploy.pm#L25
> 
> If it is so, could you please try to move the duplicated code to a dedicated method in lib/qesapdeploy.pm

Yes, similar code. Sure, I will try to move it to lib/qesapdploy.pm.

Reply:
Now that I go through all the code I see that is not going to be trivial. For example the part calling `create_instance_data` is only relevant for this test module and not for the qesap regression one.